### DISCUSSION TITLE
[MacOS]专业版无法激活，无法邮箱验证，PIN码验证按钮点击卡死

### DISCUSSION THREAD

**Discussion by imlucan** [getlantern/lantern-forum-cn#563] [Q&A] [answered]
我的购买时邮箱imlucan@icloud.com
老版本截图
![image](https://github.com/getlantern/lantern-forum-cn/assets/74130977/df12f5ce-0313-467b-be73-66b17e173b91)

## 现象
无法邮箱验证，PIN码验证按钮点击卡死。
我是今日发现以前还是网页版本连接很久连接不上，来下载新版本了。

## 我的环境
软件版本: 8.0.0 
CPU:Intel Core 
Macos:13.0.1 (22A400)

## 诉求
请协助我激活专业版

PS1:
8.0版本可以连接，无法验证与激活
7.8之后Flutter版本，同样无法验证与激活，无法连接

PS2: 
希望可以和以前一样，有个取消自动管理系统的设置选项，快速查看代理端口，目前好像是固定在了58739
因为开了之后本地App流量也会被代理（网易云音乐无法正常使用或速度很慢），白名单机制也不是太健全。
大部分情况下，我会使用SwitchOmega来管理我的代理，更灵活。
谢谢大佬！

**Comment by MaxMKMax** [comment]
@imlucan 您好. 请您也提供我支付单据. 已经帮您提交问题了. 过几天请查看邮箱或这个帖子.
我们这里可以帮您手动升级/ 或提供激活码给您. 你不介意的话可以等到下周二吗? 负责人不在.
谢谢您支持蓝灯!

**Reply by imlucan** [reply]
<img width="1024" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/74130977/68e8be9b-d818-4684-a0a7-e82cdc7026c4">

**Reply by imlucan** [reply]
您好，现在是什么进展

**Reply by MaxMKMax** [reply]
@imlucan 您好. 这是您的激活码:
CYBKQ-RDJ2F-3VFMM-8FXTD-GWGHH

不好意思让您久等了!

**Comment by MaxMKMax** [comment, marked as answer]
@imlucan 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by imlucan** [reply]
我遇到了另外一个问题，我的另一台windows 11 电脑，同样也是这个版本的flutter，无法使用Link Pin激活，可以解决么

**Comment by woodballhead** [comment]
我是注册用户wuyiwen798@163.com，但是我用用户名和注册码登录时却遇到了问题？
